### ISSUE TITLE
DMN - Ogres Idle Behavior

### ISSUE THREAD

**Issue by Kurthos** [KurthosWoW/TrinityCore#122]
Ogre Brutes and Mage-Lords should sometimes decide to take a seat (/sit) animation. Currently they are always standing. They should also randomly pace 1-2 yards periodically (This is a common blizzlike mob behavior).

**Comment by HelloKitty** [comment]
Interesting, might have to make a creature script for them then. Is there a video showing how often they do this sort of thing? I don't see anything about this on Nost I don't think. Unless @natnat sees something.

**Comment by natnat201193** [comment]
well most servers are still missing the little details like this, including nost :p ill see what they are doing currently on retail, dont think its been changed much in that reguard. as for the random movement, i get the feeling not all of them are suppose to do that. because there are packs of them talking to each other, would be a little weird for those packs to all be randomly moving and doing the talking animation.

**Comment by Kurthos** [comment]
https://youtu.be/r1Ff9kjivRg?t=13s 

You'll see an ogre sit down. Narrator talks about using distract so the Ogre doesnt randomly move.

At https://youtu.be/r1Ff9kjivRg?t=34s You can see a dog kind of turn around while staying in place.

At https://youtu.be/r1Ff9kjivRg?t=55s You can see one of the Gordok Mage-Lords using the frost bolt cast spell effect while idle.

at https://youtu.be/r1Ff9kjivRg?t=1m3s You can see another dog pacing.

**Comment by Kurthos** [comment]
Upon further searching, some NPC's have this behavior (albeit not the correct NPC's).

https://i.gyazo.com/44cc07e005060e0848431fe064958fe8.mp4

**Comment by natnat201193** [comment]
Easy to change the move type with this:
```
UPDATE `creature` SET  `MovementType`='1' WHERE  `guid`= ;
UPDATE `creature_template` SET `MovementType`='1' WHERE  `entry`= ;
```
Could probably make the npcs face the closest friendly whenever using the talk emote so they dont talk with their back turned to avoid it looking weird.

**Comment by HelloKitty** [comment]
@natnat201193 Interesting, I think Tendris does that on retail. Is he suppose to do that on 1.12.1 and 3.3.5 too?

Also, do you plan to aggergate these sql changes and commit them? No rush, but I just want to make sure we don't lose them and have to end up going into 20 different issues a week from now to deal with it.

**Comment by natnat201193** [comment]
@HelloKitty does what? random movement?  and yeah, ill put together the SQL's i commented and commit them as soon as i have set things up 👍

**Comment by HelloKitty** [comment]
Yea I was on retail yesterday and he was walking around in like a 2 yard radius back and forth it seemed. I don't know the correct behavior. Maybe Nost had it setup.

**Comment by natnat201193** [comment]
yeah, he is suppose to do that https://youtu.be/dynrgi6wxvg?t=1m11s
